kamelet-support/src/main/java/io/kaoto/backend/model/deployment/kamelet/expression/Expression.java

```diff
@@ -134,25 +170,57 @@ protected void assignProperty(final Parameter parameter) {
     @Override
     public Map<String, Object> getRepresenterProperties() {
 
-        Map<String, Object> properties = new HashMap<>();
+        Map<String, Object> properties = new LinkedHashMap<>();
```

Comment:
To keep the tests less flaky as the order is maintained.

Reply:
Is it  behavior we want at runtime too? At first sight, I would say yes, it will keep the order of elements between two save and avoid having unrelated diffs.

if it is only for the tests, the tests can be improved to ignore different orders.

Reply:
I would say yes, we want that at runtime too, at least to be predictable for the user.